lib/gui/app.js

```diff
@@ -79,17 +79,24 @@ app.run(() => {
     '\\____/ \\__\\___|_| |_|\\___|_|',
     '',
     'Interested in joining the Etcher team?',
-    'Drop us a line at join+etcher@resin.io'
+    'Drop us a line at join+etcher@resin.io',
+    '',
+    `Version = ${packageJSON.version}, Type = ${packageJSON.packageType}`
```

Comment:
Yeah, lets see how it goes, and if its useful when troubleshooting users, then we can add more info.